README.md

```diff
@@ -1,2 +1,7 @@
 # eap-cloud-galleon-pack
 Galleon feature-pack for EAP on the Cloud
```

Comment:
Do we want to say "EAP 8" instead of "EAP"?

Reply:
Fixed.